python/lsst/ts/m2gui/display/item_actuator.py

```diff
@@ -0,0 +1,147 @@
+# This file is part of ts_m2gui.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["ItemActuator"]
+
+import numpy as np
+
+from PySide2.QtCore import Qt
+from PySide2.QtWidgets import QGraphicsEllipseItem, QGraphicsTextItem
+from PySide2.QtGui import QPen
+
+from . import Gauge
+
+
+class ItemActuator(QGraphicsEllipseItem):
+    """Actuator item used in the ViewMirror class to show the actuator
+    information.
+
+    Parameters
+    ----------
+    x : `float`
+        Actuator x position on the mirror's view.
+    y : `float`
+        Actuator y position on the mirror's view.
+    diameter : `float` or `int`
+        Diameter of the actuator on the mirror's view.
+    actuator_id : `int`
+        Actuator ID.
+    alias : `str`
+        Alias of the actuator.
+    point_size : `int`
+        Point size of the text.
+
+    Attributes
+    ----------
+    label_id : `PySide2.QtWidgets.QGraphicsTextItem`
+        Label of the actuator ID.
+    """
+
+    # Maximum of the magnitude to decide the color of actuator on the mirror's
+    # view
+    MAX_MAGNITUDE = 700
+
+    def __init__(self, x, y, diameter, actuator_id, alias, point_size):
+        super().__init__(x, y, diameter, diameter)
+
+        self._acutator_id = actuator_id
+        self._alias = alias
+
+        # Magnitude to to decide the color of actuator on the mirror's view
+        self._magnitude = 0
+
+        self.label_id = QGraphicsTextItem(str(actuator_id))
```

Comment:
What about making QGraphicsTextItem child of Actuator?

```diff
diff --git a/python/lsst/ts/m2gui/display/item_actuator.py b/python/lsst/ts/m2gui/display/item_actuator.p
index d247ef2..7fbe915 100644
--- a/python/lsst/ts/m2gui/display/item_actuator.py
+++ b/python/lsst/ts/m2gui/display/item_actuator.py
@@ -68,7 +68,7 @@ class ItemActuator(QGraphicsEllipseItem):
         # Magnitude to to decide the color of actuator on the mirror's view
         self._magnitude = 0
 
-        self.label_id = QGraphicsTextItem(str(actuator_id))
+        self.label_id = QGraphicsTextItem(str(actuator_id), self)
 
         self._set_default(point_size)
 
diff --git a/python/lsst/ts/m2gui/display/view_mirror.py b/python/lsst/ts/m2gui/display/view_mirror.py
index 14c0c8b..ba10b47 100644
--- a/python/lsst/ts/m2gui/display/view_mirror.py
+++ b/python/lsst/ts/m2gui/display/view_mirror.py
@@ -89,7 +89,6 @@ class ViewMirror(QGraphicsView):
         self.actuators.append(actuator)
 
         self._mirror.addItem(actuator)
-        self._mirror.addItem(actuator.label_id)
 
     def _calculate_magnification(self, buffer_space=10):
         """Calculate the magnification. This is to re-dimensition the physical
```

Reply:
Fixed in the commit: [af8c4ea](https://github.com/lsst-ts/ts_m2gui/pull/13/commits/af8c4eada9478d63f692c5cd50c53c057e3e2b34). Thanks!